### ISSUE TITLE
Modal close button is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6862]
The Modal close button is not accessible with Windows High Contrast Mode enabled in either Firefox or Edge.

## Environment

> Operating system

Windows 10

> Browser

Firefox, Edge

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

![image](https://user-images.githubusercontent.com/40970507/93650376-f316e100-f9d3-11ea-9b4d-18f5cf6a8ca4.png)
